src/languages/es.js

```diff
@@ -165,6 +165,7 @@ export default {
         copied: '¡Copiado!',
         copyLink: 'Copiar enlace',
         copyURLToClipboard: 'Copiar URL al portapapeles',
+        copyEmailToClipboard: 'Copiar e-mail al portapapeles',
```

Comment:
Where did you get the Spanish copy?

Reply:
I did it by myself. 
Just changed the "URL" by "e-mail" from copyURLToClipboard

Reply:
The translation LGTM just change it to `email` to keep it consistent with the other uses of the `email` word in es.js